ubuntu_desktop_installer/lib/l10n/messages_it.dart

```diff
@@ -0,0 +1,45 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
+// This is a library that provides messages for a it locale. All the
+// messages from the main program should be duplicated here with the same
+// function name.
+
+// Ignore issues from commonly used lints in this file.
+// ignore_for_file:unnecessary_brace_in_string_interps, unnecessary_new
+// ignore_for_file:prefer_single_quotes,comment_references, directives_ordering
+// ignore_for_file:annotate_overrides,prefer_generic_function_type_aliases
+// ignore_for_file:unused_import, file_names
+
+import 'package:intl/intl.dart';
+import 'package:intl/message_lookup_by_library.dart';
+
+final messages = new MessageLookup();
+
+typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
+
+class MessageLookup extends MessageLookupByLibrary {
+  String get localeName => 'it';
+
+  static m0(url) => "Per istruzioni, apri questa pagina da un telefono o un altro dispositivo: <a href=\"https://${url}\">${url}</a>";
+
+  static m1(url) => "Forse vorresti leggere le <a href=\"${url}\">informazioni di rilascio</a>.";
```

Comment:
`m0` and `m1` are missing an explicit return type (`String`), see e.g. messages_fr.dart. This is [a bug in dart itself](https://github.com/dart-lang/intl_translation/issues/99), for now you will have to manually add them.